app/scripts/components/common/map/style-generators/raster-paint-layer.tsx

```diff
@@ -51,10 +49,9 @@ export function RasterPaintLayer(props: RasterPaintLayerProps) {
   const updatedTileParams = useMemo(() => {
     return {
       ...tileParams,
```

Comment:
the original sourceParams (including rescale) are passed through here in tileParams